### PR TITLE
Update fastscripts from 2.8.3 to 3.0.2

### PR DESCRIPTION
**Pull request by justin**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by danielpunkass** [comment]
Thanks @justin ! And thanks for the quick response in the other PR @neutric. Much appreciated.

**Comment by danielpunkass** [comment]
@justin do you want to update it to point to appcast3.php? I think that will keep it up to date moving forward, if I understand it.

**Comment by neutric** [comment]
It won't keep it up to date automatically but enable us to detect new versions.

**Comment by justin** [comment]
@neutric + @danielpunkass - I can update that app cast link. Separate commit/PR or just add it to this PR/commit?

**Comment by danielpunkass** [comment]
@justin I think if you just add to this PR it will re-run the check.

**Comment by neutric** [comment]
@danielpunkass `depends_on macos: ">= :sierra"` could be updated. Your sparkle file says `10.14` but on your webpage you list `10.15` as minimum OS version. Who better to decide. ;)

**Comment by danielpunkass** [comment]
@neutric Officially FastScripts requires 10.15 now but I felt bad for some folks who had to stick with 10.14 and it "mostly works" so I decided to leave it unofficially supported for now. What do you think? Probably best to make it match the Sparkle feed as far as Homebrew is concerned?

**Comment by neutric** [comment]
The point of `depends_on macos:` is to prevent a bad user experience. I'd follow your decision in this regard and go with `10.14`.

File: Casks/fastscripts.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  sha256 "fed63c476458b6a12e202ce480016912fdd55dac575f979df684d69c1bfc13a8"
 
   url "https://redsweater.com/fastscripts/FastScripts#{version.before_comma}.zip"
   name "FastScripts"
   desc "Tool for running time-saving scripts"
   homepage "https://redsweater.com/fastscripts/"
 
   livecheck do
-    url "https://redsweater.com/fastscripts/appcast2.php"
+    url "https://redsweater.com/fastscripts/appcast3.php"
```
Comment (neutric) [inline Casks/fastscripts.rb:11]: `url "https://redsweater.com/fastscripts/appcast#{version.major}.php"`

Comment (danielpunkass) [reply Casks/fastscripts.rb:11]: Ooh, good call. Save @justin the grief when I release 4.0 !

Comment (SMillerDev) [reply Casks/fastscripts.rb:11]: Not really because we would still need to know when 4.0 releases.

**Comment by bevanjkay** [comment]
@justin @danielpunkass While we're here, it would be great if someone could look at adding a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it. In a separate PR is fine.

**Comment by neutric** [comment]
Zap done in #115341.

**Comment by justin** [comment]
> Zap done in #115341.

Cheers. was planning to do it this AM. Beat me to it!

**Pull request merged**